tools/clang/test/CodeGenSPIRV/bezier.hull.hlsl2spv

```diff
@@ -114,8 +124,10 @@ BEZIER_CONTROL_POINT SubDToBezierHS(InputPatch<VS_CONTROL_POINT_OUTPUT, MAX_POIN
 // OpDecorate %out_var_TANUCORNER Patch
 // OpDecorate %out_var_TANVCORNER Patch
 // OpDecorate %out_var_TANWEIGHTS Patch
-// OpDecorate %in_var_hullEntryPointInput Location 0
-// OpDecorate %out_var_hullEntryPointOutput Location 0
+// OpDecorate %in_var_WORLDPOS Location 0
```

Comment:
Yeah, actually this is a problem we will never get it magically right for all cases since it definitely relies on the whole pipeline to know but we are only seeing one shader per time. So, just best try here.

We provide two cl options: one for assigning according to decl order, the other is alpha order. Hai requested that we should have the former as default.

Reply:
I see. Ok thanks for the explanation.